site/sfguides/src/movie_recommendation_chatbot_powered_by_kumo/movie_recommendation_chatbot_powered_by_kumo.md

```diff
@@ -0,0 +1,894 @@
+status: Draft 
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Getting Started, Data Engineering, Data Science, Snowpark Container Services, Machine Learning, Streamlit, Cortex
+
+# Personalized Movie Recommendation Chatbot Powered by Kumo in Snowflake Native App
+<!-- ------------------------ -->
+## Overview
+
+Duration: 5
+
+In this guide, you will learn how to build a personalized movie recommendation chatbot using [Kumo (as a Snowflake Native App](https://app.snowflake.com/marketplace/listing/GZTYZBN5P0/kumo-ai-kumo?search=kumo)), Snowflake’s Cortex Search Service, Cortex AI, and Streamli**t**. This chatbot enhances the streaming experience by providing personalized movie recommendations based on user viewing history and movie relationships.
+
+Specifically, you will learn how to develop a chatbot that can answer the following types of queries:
+
+- **Item-to-User Predictions** – Personalized recommendations based on a user's past viewing history (e.g., *“What should I watch next?”*).
+- **Item-to-Item Predictions** – Identifying similar movies based on shared genres, themes, or cast members (e.g., *“Is this movie similar to that one?”*).
+
+By handling both types of queries, the chatbot enhances content discovery, helping users navigate vast streaming libraries more efficiently.
+
+### What is Kumo?
+
+Kumo is an advanced ML platform that leverages [*Graph Neural Networks (GNNs)*](https://kumo.ai/research/graph-neural-networks-gnn/) to deliver predictions directly from your Snowflake relational data. Kumo ML models are powered by a GNN architecture, inspired by several academic papers in recent history. Data scientists simply need to write a [*predictive query*](https://kumo.ai/docs/predictive-query), and then the Kumo platform will automatically craft the proper neural architecture for their data. As a result of this approach, Kumo is able to achieve great predictive performance out of the box, across a wide range of challenging scenarios.
+
+Kumo simplifies the end-to-end ML process by eliminating the need for extensive data preparation, model tuning, and feature engineering, allowing data scientists to focus on generating valuable predictions and insights.
+
+To learn more, please visit [Kumo's documentation](https://docs.kumo.ai/docs/welcome-to-kumo).
+
+### What you will learn
+
+- Storing Kumo-generated movie recommendations in Snowflake
+- Setting up Cortex Search Services for Retrieval-Augmented Generation (RAG)
+- Using Cortex AI to retrieve relevant movie context and enhance chatbot responses
+- Building an interactive Streamlit UI for user interactions
+
+### What you’ll build
+
+You'll create a movie recommendation chatbot built on Snowflake, powered by Kumo predictions, handling both user-to-movie and movie-to-movie queries seamlessly.
+
+### Pre-requisites
+
+- Familiarity with [Snowflake](https://quickstarts.snowflake.com/guide/getting_started_with_snowflake/index.html#0) and a Snowflake account
```

Comment:
@effy-fang @sfc-gh-rpettus Pl remove the link to the QS guide and add a link to Snowflake login/signup page instead.